handlers/auth.go

```diff
@@ -89,6 +101,17 @@ type AuthConfig struct {
 	RoleCategories []*gaia.UserRoleCategory
 }
 
+func getPoliciesFromClaims(policyClaims interface{}) []string {
+	policies := []string{}
+	if policyClaims == nil {
```

Comment:
If this could potentially be nil inside the interface, this check will not be enough.

You'll have to also check the value in the interface to be nil or not using reflection.

```
if policyClaims == nil || reflect.ValueOf(policyClaims).IsNil() {
```